packages/graphiql-react/package.json

```diff
@@ -11,19 +11,24 @@
     "url": "https://github.com/graphql/graphiql/issues?q=issue+label:@graphiql/react"
   },
   "license": "MIT",
+  "main": "dist/index.js",
   "module": "dist/index.mjs",
+  "types": "types/index.d.ts",
   "exports": {
     "./package.json": "./package.json",
     ".": {
-      "import": "./dist/index.mjs",
-      "require": "./dist/index.js",
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs",
       "types": "./types/index.d.ts"
     },
     "./font/roboto.css": "./font/roboto.css",
     "./font/fira-code.css": "./font/fira-code.css",
-    "./dist/style.css": "./dist/style.css"
```

Comment:
I guess it worked well before?

Reply:
it did,  but this didnt work with webpack-cli serve